nbconvert/exporters/templateexporter.py

```diff
@@ -163,6 +163,31 @@ def _template_file_default(self):
     
     #Extension that the template files use.
     template_extension = Unicode(".tpl").tag(config=True, affects_environment=True)
+    
+    exclude_input = Bool(False,
```

Comment:
I chose the internal logic to be `include` and the user facing logic to be `exclude`. 

If we think of the default as what was already there, everything is included. Thus if people want something different they should have to explicitly exclude it.